lib/pf/Switch/Avaya/ERS3500.pm

```diff
@@ -0,0 +1,66 @@
+package pf::Switch::Avaya::ERS3500;
+
+=head1 NAME
+
+pf::Switch::Avaya::ERS3500 - Object oriented module to access SNMP enabled Avaya ERS 3500 switches
+
+=head1 SYNOPSIS
+
+The pf::Switch::Avaya::ERS3500 module implements an object 
+oriented interface to access SNMP enabled Avaya::ERS3500 switches.
+
+=head1 STATUS
+
+This module is currently only a placeholder, see pf::Switch::Avaya.
+
+=cut
+
+use strict;
+use warnings;
+
+use pf::Switch::constants;
+use Log::Log4perl;
+use Net::SNMP;
+
+use base ('pf::Switch::Avaya');
+
+sub supportsWiredMacAuth { return $SNMP::TRUE; }
```

Comment:
The three following methods are already defined the same way in the parent class : 
supportsWiredMacAuth
supportsWiredDot1x
supportsRadiusVoip